lib/Backend/GlobOptBlockData.cpp

```diff
@@ -482,7 +495,10 @@ GlobOptBlockData::MergeBlockData(
     if (this->capturedValues == nullptr)
     {
         this->capturedValues = fromData->capturedValues;
-        
+        if (this->capturedValues)
+        {
+            this->capturedValues->IncrementRefCount();
```

Comment:
>IncrementRefCount [](start = 34, length = 17)

Do we call delete on the fromBlock after merge?

Reply:
Depends on the dataUseCount of the fromBlock, no?